src/utils/temp_folder.test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import { cleanUpTempFolder, getTempFolder } from './temp_folder';
+import * as fs from 'fs';
+
+describe('temp folder test', () => {
+	it('temp returns a folder that exists', () => {
+		const tempFolderPath = getTempFolder();
+		expect(fs.existsSync(tempFolderPath)).to.equal(true);
+	});
+
+	it('temp path returned contains the correct subfolders', () => {
+		const tempFolderPath = getTempFolder();
+		expect(tempFolderPath).to.contains('ardrive');
```

Comment:
How about checking the os string in this test and doing a more precise expectation? e.g.
```
const expectedPathComponent = os.platform() === 'win32' ? 'ardrive-temp' : '.ardrive/temp';
expect(tempFolderPath).to.contains(expectedPathComponent);
```